src/style.css

```diff
@@ -1 +1,64 @@
 /* Styles go here */
+
+html,
+body {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  height: 100%;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  height: 60px;
+  white-space: nowrap;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  transition: color 0.3s;
+  cursor: pointer;
+}
+
+.nav__link:hover,
+.nav__link.is-active {
+  color: #00acdc;
+}
```

Comment:
Don't bind rules together
Because then you will have several places to check to get all the rules for the same element (as you have it here with the next rule)